SPECS/dracut/allow-liveos-overlay-no-user-confirmation-prompt.patch

```diff
@@ -0,0 +1,22 @@
+--- a/modules.d/90dmsquash-live/dmsquash-live-root.sh	2021-05-27 05:34:19.000000000 -0700
```

Comment:
Minor style thing, but I tend to prefer using `git format-patch...` to create these instead of `git diff`. The patch is the same but we can include author, commit message, etc.

This is small enough to not really matter though, very clear what its trying to do.

Reply:
thx Daniel - I've re-generated the patch using `git format-patch` - much better.